task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -180,10 +180,18 @@ spec:
           TESTPASSED=false
         fi
 
-        # examines the BASE_IMAGE tag to derive the target OCP version
+        # examines the base_image_name tag to derive the target OCP version
         # assumes this is in the form
         # image-path:[v]major-digits.minor-digits[@sha...]
-        OCP_VER_FROM_BASE=$(echo "${BASE_IMAGE}" | sed -e "s/@.*$//" -e "s/^.*://")   # strips hash first due to greedy match
+        status=0
+        base_image_name=$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" ../inspect-image/image_inspect.json) || status=$?
+        if [ $status -ne 0 ]; then
+          echo "Could not get labels from inspect-image/image_inspect.json. Make sure file exists and it contains this label: org.opencontainers.image.base.name"
+          TEST_OUTPUT="$(make_result_json -r ERROR)"
+          echo "${TEST_OUTPUT}" | tee "$(results.TEST_OUTPUT.path)"
+          exit 0
+        fi
+        OCP_VER_FROM_BASE=$(echo "${base_image_name}" | sed -e "s/@.*$//" -e "s/^.*://")   # strips hash first due to greedy match
```

Comment:
would it be less "greedy" to replace the regex `(.*)`  with  `(.*?)` , wdyt?

Reply:
Tbh I didn't want to change the original implementation of the line since it's not in the scope of this change.
I only changed the variable that's used.

Reply:
fair enough, 
thanks